src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -1,6 +1,6 @@
 package seedu.address.logic.commands;
 
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.PersonContainsKeywordsPredicate;
```

Comment:
HAHA just wondering, why did you have to change it from `Name...` to `Person...`?

Reply:
Oh ok never mind, I realised why LOL.

Reply:
haha yup yup